test/utils.test.ts

```diff
@@ -744,6 +744,32 @@ describe('#organizeArguments()', () => {
 			utils(settings).organizeArguments();
 		}).toThrow();
 	});
+
+	test('Complains about unexpected pass-through arguments ', () => {
+		const settings = {
+			...settingsPizzaOrdering,
+			arguments: ['order', 'dine-in', '--', '--pass-through-flag', 'pass-through-option=true', 'pass-through-data'],
+		};
+
+		expect(() => {
+			utils(settings).organizeArguments();
+		}).toThrow();
+	});
+
+	test('Handles having pass-through arguments', () => {
+		const settings = {
+			...settingsPizzaOrdering,
+			arguments: ['order', 'to-go', '--', '--pass-through-flag', 'pass-through-option=true', 'pass-through-data'],
+		};
+
+		expect(utils(settings).organizeArguments()).toStrictEqual({
+			command: 'order to-go',
+			flags: [],
+			options: [],
+			passThrough: ['--pass-through-flag', 'pass-through-option=true', 'pass-through-data'],
```

Comment:
Same reasoning as above:

```suggestion
			passThrough: ['--pass-through-flag', 'pass-through-option=value', 'pass-through-data'],
```